src/query/interpreter.cpp

```diff
@@ -3790,6 +3791,9 @@ void RunTriggersIndividually(const utils::SkipList<Trigger> &triggers, Interpret
     auto storage_acc = interpreter_context->db->Access();
     DbAccessor db_accessor{storage_acc.get()};
 
+    // On-disk storage removes all Vertex/Edge Accessors after DbAccessor is destroyed.
```

Comment:
But doesn't this happen also for InMemory storage? How is that different for OnDisk?